samples/.eslintrc.yml

```diff
@@ -1,3 +1,4 @@
 ---
 rules:
   no-console: off
+  node/no-missing-require: off
```

Comment:
Is this intentional?

Reply:
Yeah, sadly.  Otherwise when you run `npm run fix` locally, it throws all kinds of errors unless you've cd'd into samples and run `npm i`